compiler/rustc_codegen_llvm/src/gotoc/cbmc/goto_program/expr.rs

```diff
@@ -698,6 +698,27 @@ impl Expr {
         Expr::struct_expr_with_explicit_padding(typ, fields, values)
     }
 
+    /// Struct initializer  
+    /// `struct foo the_foo = >>> {field1, padding2, field3, ... } <<<`
+    /// Note that padding fields should be explicitiy given.
```

Comment:
Can the first line give a little more context on how this is used?

+ Nit: "explicitly"

Reply:
The dangers of copy-pasting existing comments...

Fixed!